balrogscript/src/balrogscript/apiv2.py

```diff
@@ -0,0 +1,42 @@
+# This should live in balrogclient in the balrog repo, placed here to facilitate easy testing
```

Comment:
Thanks for writing most of this! I'll tidy it up a bit and get it into `balrogclient`.